src/platform-implementation-js/main.ts

```diff
@@ -9,29 +16,27 @@ if (!global.__InboxSDKImpLoader) {
     '[role=banner] div[aria-label] div div a[href^="https://myaccount.google."], [role=banner]+div div[aria-label] div div a[href^="https://myaccount.google."]'
   );
 
-  var Kefir = require('kefir');
-
   var onready = new Promise<void>((resolve) => {
     if (
       document.readyState === 'complete' ||
       document.readyState === 'interactive'
     ) {
       resolve();
     } else {
-      resolve(
-        Kefir.merge([
-          Kefir.fromEvents(document, 'DOMContentLoaded'),
-          Kefir.fromEvents(window, 'load'),
-        ])
-          .take(1)
-          .map(() => null)
-          .toPromise()
+      const fromEvent = async (event: 'DOMContentLoaded' | 'load') =>
```

Comment:
i'm out of context what is this refactoring for, but flagging this behaviour change, when before first event that happened would trigger `resolve` is changed to wait untill all two events get triggered and then execute resolve. also i believe Kefir.fromEvents will unsubscribe when the take(1) is finished, but with new code it will never unsubscribe.

Reply:
[67259e3](https://github.com/InboxSDK/InboxSDK/pull/971/commits/67259e38c2c2900ea48997f7d6146a2a392d6e5b)